index.bs

```diff
@@ -1404,13 +1404,7 @@ The {{MLGraphBuilder}} interface defines a set of operations as identified by th
 <script type=idl>
 typedef record<DOMString, MLOperand> MLNamedOperands;
 
-dictionary MLBufferResourceView {
-  required GPUBuffer resource;
-  unsigned long long offset = 0;
-  unsigned long long size;
-};
-
-typedef (ArrayBufferView or MLBufferResourceView) MLBufferView;
+typedef ArrayBufferView MLBufferView;
```

Comment:
Follow-up question: Do we still need `MLBufferView`?

Reply:
> I suppose we won't need it.

I think it depends if `MLBuffer` will be allowed to combine input/output. If so, the WebNN developer could create one large `MLBuffer` to hold inputs and outputs then dispatch them via `MLBufferView`.

Reply:
@a-sully wrote

> I'll put up an issue to discuss

SGTM, thanks!

Reply:
SGTM, @a-sully.

Reply:
Filed #560. I'll put up a follow-up PR to remove the `MLBufferView` typedef (and another to remove it from the Model Loader API specification, I suppose)